lib/rules/no-duplicate-selectors/index.js

```diff
@@ -64,13 +73,11 @@ const rule = (primary, secondaryOptions) => {
 			const resolvedSelectorList = [
 				...new Set(
 					ruleNode.selectors.flatMap((selector) => {
-						return isStandardSyntaxSelector(selector)
-							? resolvedNestedSelector(selector, ruleNode)
-							: [];
+						return shouldParseSelector(selector) ? resolvedNestedSelector(selector, ruleNode) : [];
 					}),
 				),
 			];
```

Comment:
Please see the doc: https://stylelint.io/user-guide/rules/about/

> The built-in rules: apply to standard CSS syntax only

It's almost impossible that we will support and maintain many CSS-like syntaxes. So we've just ignored non-standard syntax (can parse them partially, though).

3rd-party plugins should support such syntaxes, instead of the Stylelint built-in rules.